crates/bevy_solari/src/lib.rs

```diff
@@ -46,7 +46,7 @@ impl SolariPlugin {
             | WgpuFeatures::EXPERIMENTAL_RAY_QUERY
             | WgpuFeatures::BUFFER_BINDING_ARRAY
             | WgpuFeatures::TEXTURE_BINDING_ARRAY
-            | WgpuFeatures::UNIFORM_BUFFER_AND_STORAGE_TEXTURE_ARRAY_NON_UNIFORM_INDEXING
+            | WgpuFeatures::STORAGE_TEXTURE_ARRAY_NON_UNIFORM_INDEXING
```

Comment:
I think you can remove this line outright. The only thing I need binding arrays of are:
1. Storage buffers
2. Sampled textures
3. Samplers